src/patternfly/components/DataList/data-list.scss

```diff
@@ -238,6 +249,57 @@
     box-shadow: var(--pf-c-data-list__item--m-selected--BoxShadow);
   }
 
+  // stylelint-disable
+  &.pf-m-draggable {
+    position: relative;
+
+    .pf-c-data-list__item-draggable-button {
+      padding-right: var(--pf-c-data-list__item--m-draggable__draggable-button--PaddingRight);
+      padding-left: var(--pf-c-data-list__item--m-draggable__draggable-button--PaddingLeft);
+      margin-top: var(--pf-c-data-list__item--m-draggable__draggable-button--MarginTop);
+      margin-left: var(--pf-c-data-list__item--m-draggable__draggable-button--MarginLeft);
+      border: 0;
+
+      &:hover {
+        cursor: grab;
+
+        --pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color: var(--pf-c-data-list__item--m-draggable__draggable-button--hover__draggable-icon--Color);
+      }
+
+      &:active {
+        cursor: grabbing;
+      }
+
+      &.pf-m-disabled {
+        pointer-events: none;
+        cursor: none;
+
+        --pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color: var(--pf-c-data-list__item--m-draggable__draggable-button--m-disabled__draggable-icon--Color);
+      }
+
+      .pf-c-data-list__item-draggable-icon {
+        color: var(--pf-c-data-list__item--m-draggable__draggable-button__draggable-icon--Color);
```

Comment:
you could set this on the `.pf-c-data-list__item-draggable-button` element. I doubt we need this element since it's basically just a plain button, but we could keep it, too, in case we need it.

Reply:
yeah I was thinking of keeping it for now in case anything changes with the interaction of the draggable icon